### PR TITLE
Add  gabrielwen  & zhenghuiwang to OWNERS

### PR DESCRIPTION
**Pull request by zhenghuiwang**
The ONWERS file https://github.com/kubeflow/testing/blob/master/py/kubeflow/OWNERS doesn't work. So add us to the top level OWNERS.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/389)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,3 +6,5 @@ approvers:
   - kunmingg
   - richardsliu
   - zabbasi
+  - gabrielwen
```
Comment (jlewi) [inline OWNERS]: Alphabetical order please

Comment (zhenghuiwang) [reply OWNERS]: Done.

**Pull request merged**